### PR TITLE
This is for windows machines

### LINKED ISSUE #34: Directory creation in Windows: Error
makedir from linux uses '/' whereas Windows uses '\\' for paths in the creation of files and folders in a directory.
So instead of hard coding it for Windows and Linux separately, We'd like to have os.join to detect the os and choose the right option in the python script.

### PR DESCRIPTION
**Pull request by HardipinderS**
issue fix #34

### CODE REVIEW

**Comment by HardipinderS** [comment]
this is for windows machiens

File: crawler.py [hunk 1]
```diff
@@ -30,8 +30,13 @@ def mkdir(path):
     if os.makedirs(path):
         log.info('created {}'.format(path))
         
-
-PREFIX = '{}/{}'.format(os.path.dirname(__file__), 'data')
+"""
+Comment here
+"""
+# PREFIX = '{}/{}'.format(os.path.dirname(__file__), 'data')
```
Comment (meain) [inline crawler.py]: could you remove the comments here

File: crawler.py [hunk 2]
```diff
@@ -59,20 +64,21 @@ def __init__(self, root_url, root_dir='', prefix=PREFIX):
         
         self.ROOT_URL = root_url
         if root_dir:
-            root_dir = '/'.join(root_dir.split('/')[-2:])
+            print('hello', root_dir)
```
Comment (meain) [inline crawler.py]: remove this print

File: crawler.py [hunk 3]
```diff
@@ -59,20 +64,21 @@ def __init__(self, root_url, root_dir='', prefix=PREFIX):
...
+            root_dir = (os.sep).join(root_dir.split(os.sep)[-2:])
             root_dir = root_dir.replace('crawler-', '').replace('.py', '')
```
Comment (meain) [inline crawler.py]: might be rewriting the change you made above, could you check and remove it

**Comment by adamshamsudeen** [comment]
@HardipinderS thanks a lot. :)

**Pull request merged**